regression/pages/studio/grading_studio.py

```diff
@@ -19,14 +17,14 @@ def url(self):
         """
         Construct a URL to the page within the course.
         """
-        course_key = get_course_key(self.course_info)
-        url = "/".join(
-            [BASE_URL, self.url_path, urllib.quote_plus(unicode(course_key))])
-        return url if url[-1] is '/' else url + '/'
+        url = get_url(self.url_path, self.course_info)
```

Comment:
Not a must, but you can compress this into one line:

`return get_url(self.url_path, self.course_info)`